### DISCUSSION TITLE
Authentication error: you cannot auth exceed once in 60 seconds

### DISCUSSION THREAD

**Discussion by MattRoe84** [milo526/homebridge-tuya-web#349] [Q&A]
Hi, looks like this issue is back. I've had the pluggin up and running fine with no issue for some time and a couple of nights back night im now getting this error. [TuyaWebPlatform] Cannot acquire token, waiting 65 seconds. [TuyaWebPlatform] Retrying authentication after previous error.
[10/16/2021, 8:48:58 AM] [TuyaWebPlatform] Authentication error: you cannot auth exceed once in 60 seconds

**Comment by rkhirata** [comment]
Any news on this?
